2019/src/i18n/en.ts

```diff
@@ -0,0 +1,44 @@
+export const en = {
+  translation: {
+    "about.description": "TODO: about.description",
+    "schedule.description": "TODO: schedule.description",
+    "speakers.description": "",
+    "sponsors.description": "",
+    timetable: "Timetable",
+    "timetable.description": "",
+    backToTop: "TOP",
+    siteName: "JSConf JP",
+    description:
+      "jsconf.jp is a JavaScript festival in Japan powered by Japan Node.js Association. This is the first event of jsconf in Japan. We would love to become a bridge between Japanese Web Developers and International Web Developers.",
+    festivalPeriod: "November 30th, December 1st 2019",
+    aboutUs: "About us",
+    guestSpeakers: "Guest speakers",
+    goToGuests: "CHECK ALL SPEAKERS",
+    schedule: "schedule",
+    day1: "Day 1",
+    day2: "Day 2",
+    tickets: "Tickets",
+    ticketsDescription: "Tickets will be available from October 1th.",
```

Comment:
a minor typo here. should be **October 1st**.

By the way, is this date confirmed?
Will the ticket price be announced before that day?

Looking forward to joining the event!